TestCases/CreatePost/tc1.robot

```diff
@@ -0,0 +1,67 @@
+*** Settings ***
+Library           SeleniumLibrary
+
+*** Variables ***
+${BROWSER}    Chrome
+${URL}    https://app-stage.thearkofrevival.com
+${EMAIL}    admin@justomerchantz.com
+${PASS}    P@ssword0123
+${POST_DESC_TEST}    automation generated post tester2
+${EMAIL_LABEL_TEST}    Email
+${PASS_LABEL_TEST}    Password
+${EXPECTED_MESSAGE}    Login success 😃!
+
+*** Test Cases ***
```

Comment:
keep only test case under test case section and execute the commands by invoking keywords.
For example:
*** Test Cases ***
Verify that Admin or Global admin can login
      Launch Browser
*** Keywords ***
Launch Browser
      Command to perform browser action

For initial phase of learning this is good.